packages/connect/src/core/onCallFirmwareUpdate.ts

```diff
@@ -180,6 +180,66 @@ const waitForReconnectedDevice = async (
     return reconnectedDevice;
 };
 
+const getRebootMethod = async ({
+    deviceList,
+    device,
+    log,
+}: {
+    deviceList: DeviceList;
+    device: Device;
+    log: Log;
+}) => {
+    let method: ReconnectParams['method'] = 'wait';
+
+    // not a bluetooth device
+    if (device.bluetoothProps) {
```

Comment:
This may be unwanted.
```suggestion
    if (!device.bluetoothProps) {
```

Reply:
cool, thx, not intentional